dids/src/main/kotlin/web5/sdk/dids/methods/dht/DhtClient.kt

```diff
@@ -208,6 +205,15 @@ internal class DhtClient(
       }
     }
 
+    private fun bencode(bs: ByteArray): ByteArray {
+      val out = ByteArrayOutputStream()
+      val l = bs.size.toString()
+      out.write(l.toByteArray(charset("UTF-8")))
+      out.write(58)
+      out.write(bs)
+      return out.toByteArray()
+    }
+
```

Comment:
can we add context for what this method does? maybe https://en.wikipedia.org/wiki/Bencode and include a test?

Reply:
Added docs and tests.